alf/utils/tensor_utils.py

```diff
@@ -103,7 +103,7 @@ def _var(x):
         return var
 
     vary = _var(y)
-    r = 1 - _var(y - ypred) / (vary + 1e-30)
+    r = torch.where(vary == 0, vary, 1 - _var(y - ypred) / (vary + 1e-30))
```

Comment:
Why setting the ratio to 0 if vary==0?

Reply:
So that it doesn't blow up the tensorboard graph.